azure-pipelines.yml

```diff
@@ -208,7 +208,7 @@ jobs:
       artifactName: metadata
 
   - task: PipAuthenticate@1
-    condition: eq(variables['Build.SourceBranch'], 'refs/heads/release')
+    condition: startsWith(variables['Build.SourceBranch'], 'refs/heads/release')
```

Comment:
This is a functionality change so not a backport.

Reply:
This pr is the same as #30554, it backports the CI changes to old version.